robottelo/cli/factory.py

```diff
@@ -1820,26 +1820,28 @@ def setup_org_for_a_rh_repo(options=None):
         raise CLIFactoryError('Failed to upload manifest')
     # Enable repo from Repository Set
     result = RepositorySet.enable({
-        u'name': options.get('repository-set', None),
+        u'name': options.get('repository-set'),
         u'organization-id': org_id,
-        u'product': options.get('product', None),
-        u'releasever': '7Server',
+        u'product': options.get('product'),
+        u'releasever': options.get('releasever'),
```

Comment:
If this is not specified None will be returned and the option will not be passed to the built hammer command.